nuklear.h

```diff
@@ -3730,7 +3730,10 @@ NK_API int nk_stricmp(const char *s1, const char *s2);
 NK_API int nk_stricmpn(const char *s1, const char *s2, int n);
 NK_API int nk_strtoi(const char *str, const char **endptr);
 NK_API float nk_strtof(const char *str, const char **endptr);
+#ifndef NK_STRTOD
+#define NK_STRTOD nk_strtod
```

Comment:
`nk_strtod` only ifdefs out the declarations while `nk_memcpy` ifdefs out both the declaration and definition. Should they all ifdef out the definition or?

Reply:
Ah of course - sorry - will fix it.

Reply:
Fixed now - thanks for reporting.